### PR TITLE
Fix ubuntu 2004 reposync issue and 2204 reposync issue

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?

Fix ubuntu synchronization issues during BV uyuni reposync stage.

## Errors
```bash
uyuni-bv-master-server:~ # mgrctl exec -i 'spacewalk-common-channels -u admin -p admin --list' | grep ubuntu-2204-pool-amd64-uyuni
 ubuntu-2204-pool-amd64-uyuni: amd64-deb
```

image
image

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite
- No tests: already covered

- [x] **DONE**

## Links

Port :

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: testsuite/features/support/constants.rb [hunk 1]
```diff
@@ -1326,7 +1326,7 @@
       ],
     'ubuntu-2404' =>
       %w[
-        ubuntu-24.04-pool-amd64-uyuni
+        ubuntu-2404-pool-amd64-uyuni
```
Comment (Bischoff) [inline testsuite/features/support/constants.rb:1329]: below (line 1646) and above (line 448) it has a dot

**Review by Bischoff — CHANGES_REQUESTED** [review]
spotted inconsistency

File: testsuite/features/support/constants.rb (hunk 1 above)
Comment (mcalmer) [reply testsuite/features/support/constants.rb:1329]: AFAIK this inconsistency is real and exists. Please check spacewalk-common-channels.ini .
The Community wanted it

Comment (Bischoff) [reply testsuite/features/support/constants.rb:1329]: I'm not speaking about the inconsistency between `2204` and `24.04`.

I'm speaking about the inconsistency between `ubuntu-2404-pool-amd64-uyuni` and `ubuntu-24.04-pool-amd64-uyuni` in this file.

Comment (maximenoel8) [reply testsuite/features/support/constants.rb:1329]: So this new name for 2404 is correct @mcalmer ?

Comment (maximenoel8) [reply testsuite/features/support/constants.rb:1329]: It's the value from `spacewalk-common-channel` for the others, in the testsuite, they have no dots.

Comment (maximenoel8) [reply testsuite/features/support/constants.rb:1329]: In spacewalk-commons-channel.ini I can see 20.04, 22.04 and 24.04 has the same description.

Comment (Bischoff) [reply testsuite/features/support/constants.rb:1329]: if all 3 have no dots in spacewalk-common-channels.ini, then current modified PR is correct. Approving.

**Review by Bischoff — APPROVED** [review]
see my comment

**Review by Bischoff — APPROVED** [review]

**Pull request merged**